pages/donate/index.tsx

```diff
@@ -0,0 +1,55 @@
+import { Box, Center, Container, Flex, Link, Text } from '@chakra-ui/react';
+
+const Donate: React.FC = () => {
+  return (
+    <>
+      <Flex gap={'150px'} flexDir={'column'}>
+        <Container maxW={'container.lg'} mb="100px">
+          <Flex gap={'60px'} flexDir="column">
+            <div data-cy="donate-top-message" id={'donate-top-message'}>
+              <Flex flexDir="column" gap={'15px'}>
+                <Center>
+                  <Box fontSize="5xl">Any amount helps us reach our goal!</Box>
+                </Center>
+              </Flex>
+            </div>
+            <div data-cy="donate-top-submessage" id={'donate-top-submessage'}>
+              <Flex flexDir="column" gap={'15px'}>
+                <Center>
```

Comment:
Why do we need a Center component here? Can we use justify-content or align-items in the Flex instead?

Reply:
also this text looks off center in the picture, not sure if that was fixed

Reply:
not working, this is how Chakra seems to handle alignment